src/industry_cmd.cpp

```diff
@@ -1824,14 +1815,14 @@ static void DoCreateNewIndustry(Industry *i, TileIndex tile, IndustryType type,
 	if (_generating_world) {
 		if (HasBit(indspec->callback_mask, CBM_IND_PRODUCTION_256_TICKS)) {
 			IndustryProductionCallback(i, 1);
-			for (size_t ci = 0; ci < lengthof(i->last_month_production); ci++) {
-				i->last_month_production[ci] = i->produced_cargo_waiting[ci] * 8;
-				i->produced_cargo_waiting[ci] = 0;
+			for (auto &p : i->produced) {
+				p.history[1].production = p.waiting * 8;
```

Comment:
Would it make sense to have something like `static const int LAST_MONTH = 1;` (and `CURRENT_MONTH = 0`) to give more intrinsic meaning to this statement?